package.json

```diff
@@ -27,7 +27,10 @@
     "prebuild": "npm run doc-listing-inject",
     "build": "node scripts/build.js",
     "postbuild": "rm -rf examples/node_modules && mkdir examples/node_modules && cp -r node_modules/@webcomponents node_modules/requirejs examples/node_modules/",
-    "clean": "rm -rf node_modules package-lock.json elements/*/{node_modules,package-lock.json,_temp} test/vrt-snapshots",
+    "clean": "rm -rf node_modules elements/*/{node_modules,package-lock.json}",
+    "clean:components": "rm -rf elements/*/{dist,_temp}",
+    "clean:e2e": "rm -rf test/vrt-snapshots",
+	"clean:all": "npm-run-all --parallel clean clean:components clean:e2e",
```

Comment:
Indentation looks a bit weird but I like 👍

Reply:
Yeah why does it look weird

Reply:
I think it's something about github.  It's fine in the file itself